test/unit/client/test_build_runner.py

```diff
@@ -1,42 +1,94 @@
 from requests.models import Response
-from unittest.mock import MagicMock, Mock
+from unittest.mock import MagicMock, Mock, patch, call
 
+from app.client import build_runner
 from app.client.build_runner import BuildRunner
 from test.framework.base_unit_test_case import BaseUnitTestCase
 
 
 class TestBuildRunner(BaseUnitTestCase):
 
-    def mock_runner_with_status_response(self, response):
-        runner = BuildRunner('url', {}, 'mellon')
-        start_response = Response()
-        start_response.json = MagicMock(return_value={"build_id": "1"})
-        runner._network.post = MagicMock(return_value=start_response)
-        status_response = Response()
-        status_response.json = MagicMock(return_value=response)
-        runner._network.get = MagicMock(return_value=status_response)
-        runner._download_and_extract_results = MagicMock()
-        return runner
+    def _create_build_runner_instance(self, response):
+        with patch(build_runner.__name__ + '.get_logger') as mock_get_logger:
```

Comment:
We have a patch helper in `BaseUnitTestCase` that I feel creates a cleaner syntax. So you can do:
```python
mock_get_logger = self.patch('app.client.build_runner.get_logger')
...
```